R/PipeOp.R

```diff
@@ -191,8 +191,9 @@ PipeOp = R6Class("PipeOp",
     input = NULL,
     output = NULL,
     .result = NULL,
+    tags = NULL,
 
-    initialize = function(id, param_set = ParamSet$new(), param_vals = list(), input, output, packages = character(0)) {
+    initialize = function(id, param_set = ParamSet$new(), param_vals = list(), input, output, packages = character(0), tags = character(0)) {
```

Comment:
could you add documentation to the `tags` parameter of the `new()` function of this class?